qualtran/_infra/composite_bloq.py

```diff
@@ -771,7 +771,9 @@ def add_register(
         return None
 
     @classmethod
-    def from_signature(cls, signature: Signature, add_registers_allowed=False):
+    def from_signature(
```

Comment:
does mypy / pylint not raise an error without a type hint?

Reply:
it does not